macaw-ppc-symbolic/src/Data/Macaw/PPC/Symbolic/Functions.hs

```diff
@@ -314,6 +314,8 @@ funcSemantics sf pf s =
       fval <- lookupApplySymFun sym sf ("vec_" ++ name) C.knownRepr args C.knownRepr
       ptrVal <- LL.llvmPointer_bv sym fval
       return (ptrVal, s)
+    MP.PPCSyscall {} ->
+      P.panic P.PPC "funcSemantics" ["The PPC syscall primitive should be eliminated and replaced by a handle lookup"]
```

Comment:
Should we just remove this constructor? Or is it necessary for this to happen after translation into a Macaw-PPC CFG?

Reply:
It's still necessary to have the `PPCSyscall` primitive function, as it exists for some (but not all) parts of the `macaw-symbolic` translation pipeline. See [`Note [Syscalls]`](https://github.com/GaloisInc/macaw/blob/83d3907054181b17be365bdea9abb25edd48aae3/x86_symbolic/src/Data/Macaw/X86/Symbolic.hs#L396-L424).